lib/sdk/io/file.js

```diff
@@ -129,20 +129,30 @@ exports.list = function list(path) {
 };
 
 exports.open = function open(filename, mode) {
-  var file = MozFile(filename);
-  if (typeof(mode) !== "string")
+  const file = MozFile(filename);
+  if (typeof (mode) !== "string") {
     mode = "";
+  }
```

Comment:
It's true this existing code doesn't match the style guide perfectly but I'd rather leave it alone than lose blame at this point.